.github/workflows/ci.yaml

```diff
@@ -17,11 +17,11 @@ jobs:
       - run: npx prettier . --check
       - run: yarn
       - run: yarn compile-ballerina-core
-  ballerina-core:
+  ballerina-forms:
```

Comment:
fyi @PawelStadnicki , when CI is changed, one must also update the settings in GH, otherwise:
- new PRs will be stuck expecting the old checks to run (e.g. https://github.com/Ballerina-Org/ballerina/pull/404)
- the new checks you added won't actually be required (one can merge without CI passing)